build.sbt

```diff
@@ -217,7 +221,8 @@ lazy val specs2 = projectMatrix
     name := "specs2",
     testFrameworks := Seq(new TestFramework("weaver.framework.CatsEffect")),
     libraryDependencies ++= Seq(
-      "org.specs2" %%% "specs2-matcher" % "4.10.5"
+      ("org.specs2" %%% "specs2-matcher" % "4.10.5").withDottyCompat(
```

Comment:
I'm not seeing a 3.0.0-M3 release yet, so this might have to become onlyScala2 for the time being.

Reply:
I'm fine with that

Reply:
i.e. with making it scala 2 only?

Reply:
> i.e. with making it scala 2 only?

yes